### ISSUE TITLE
Connection loss after idle period to always-on peer (server): restart fixes it

### ISSUE THREAD

**Issue by aral** [beakerbrowser/beaker#1057]
Operation System: macOS v. 10.13.5 (17F77)
Beaker version: v0.7.11 

__Screen capture of issue:__ https://vimeo.com/277244303

__Video shows:__ A new Window being opened in a beaker browser session that has been idle for a while (I accidentally closed the previous window, which is where I noticed the issue, before starting to record. Opening a new window does not fix the issue). You can see that the HTTPS version works. Clicking the P2P version button leads to a timeout. Following the timeout, I restart Beaker and both the HTTPS and P2P versions load correctly.

I’ve noticed this several times now: after an idle period, Beaker Browser can lose connection to an always-on peer (in my case, my web site, at dat://ar.al). A restart of Beaker always fixes it.

## beaker://debug-log

```
…
Wed, 27 Jun 2018 09:44:03 GMT dns-discovery MDNS query 192.168.2.13:5353 3Q 5A +1
Wed, 27 Jun 2018 09:44:03 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:03 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:03 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:03 GMT dns-discovery Invalid ID in answer, discarding { name: '.', domain: '', host: '192.168.2.13', port: 5353 }
Wed, 27 Jun 2018 09:44:06 GMT dns-discovery MDNS query 192.168.2.13:5353 3Q 5A +1
Wed, 27 Jun 2018 09:44:06 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:06 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:06 GMT dns-discovery Replying announce via PTR  to 192.168.2.13:5353
Wed, 27 Jun 2018 09:44:06 GMT dns-discovery Invalid ID in answer, discarding { name: '.', domain: '', host: '192.168.2.13', port: 5353 }
Wed, 27 Jun 2018 09:44:11 GMT dns-discovery MDNS query 192.168.2.109:5353 4Q 0A +0
Wed, 27 Jun 2018 09:44:11 GMT dns-discovery Replying announce via SRV  to 192.168.2.109:5353
Wed, 27 Jun 2018 09:44:11 GMT dns-discovery Replying announce via SRV  to 192.168.2.109:5353
Wed, 27 Jun 2018 09:44:11 GMT dns-discovery Replying announce via SRV  to 192.168.2.109:5353
Wed, 27 Jun 2018 09:44:11 GMT dns-discovery Replying announce via SRV  to 192.168.2.109:5353
Wed, 27 Jun 2018 09:44:20 GMT dns-discovery MDNS query 192.168.2.109:5353 2Q 0A +0
Wed, 27 Jun 2018 09:44:20 GMT dns-discovery Replying announce via SRV  to 192.168.2.109:5353
```

**Comment by pfrazee** [comment]
This might be solved in the 0.8.0 pre-releases, but it's something we've struggled with intermittently. I'll leave this open and we can watch for it when 0.8.0 "final" comes out.

**Comment by pfrazee** [comment]
(Thanks for reporting)
